css/src/atomics/overflow.scss

```diff
@@ -2,8 +2,16 @@
 	overflow: hidden;
 }
 
+.overflow-x-hidden {
```

Comment:
not sure why it missed in this gile, but atomics should come with `!important` flag